docs: add information about admin-only organization rename permissions (#3139)
neondatabase/website: content/docs/manage/orgs-manage.md

  ![organization members](/docs/manage/orgs_members_kebab.png 'no-border')

## Rename an organization

> Only Admins can rename an organization from the **Settings** page under the **General information** section. The organization name change applies globally and will appear this way for all members of the organization.

Reviewer: ```suggestion
Only Admins can rename an organization. Go to the **Settings** page under **General information**. Changing the organization name applies globally—the new name will appear for everyone in the organization.
```
